spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -126,4 +126,34 @@
       end
     end
   end
+
+  describe 'GET#show' do
+    context 'given a valid survey ID' do
+      it 'returns success status' do
+        user = api_sign_in
+        Fabricate(:keyword, user: user, id: 1)
+        get :show, params: { id: 1 }
```

Comment:
We can avoid setting hardcoded ID param. Same for others tests in this file

```suggestion
        keyword = Fabricate(:keyword, user: user, id: 1)
        get :show, params: { id: keyword.id }
```

Reply:
https://github.com/suho/google-search-ruby/pull/56/commits/94a0f67a4744e5efbb7f23d7b1f076b5e191f3c5